MODEL_SETUP.jl

```diff
@@ -375,6 +377,12 @@ elseif planet=="Venus"
                                     :O2=>Dict("n"=>[3e-3*ntot_at_lowerbdy, NaN], "f"=>[NaN, 0.]),
                                     :N2=>Dict("n"=>[0.032*ntot_at_lowerbdy, NaN]),
 
+                                    #Krasnopolsky, 2010a: this was 400ppb at 74km in altitude, and the actual number is likely lower (is either 4.0E-7, or 4.8E-7 depending on the calculation); and according to Zhang 2012 it is 3.66e-7
+                                    :HCl=>Dict("n"=>[3.66e-7 * ntot_at_lowerbdy, NaN]),
+
+                                    #Denis A. Belyaev 2012: this was 0.1 ppmv at 165–170 K to 0.5–1 ppmv at 190–192 K; It said 0.1ppm was related to the most common temperature reading so I went with that (this is either 1E-7 or 6.79E-8 depending on the calculation)
+                                    :SO2=>Dict("n"=>[1.0E-7 * ntot_at_lowerbdy, NaN]),
```

Comment:
Is this one using the model fit they did with just SO2 or with SO2 and SO together? I noticed those are pretty different